src/Tribe/Tickets.php

```diff
@@ -2490,16 +2490,29 @@ public function send_tickets_email_for_attendee( $to, $tickets, $args = [] ) {
 			$sent = $send_callback( $to, $subject, $content, $headers, $attachments );
 
 			// Handle marking the attendee ticket email as being sent.
-			if ( $sent && ! empty( $ticket_sent_meta_key ) ) {
+			if ( $sent ) {
 				// Mark attendee ticket email as being sent for each attendee ticket.
 				foreach ( $tickets as $attendee ) {
-					update_post_meta( $attendee['attendee_id'], $ticket_sent_meta_key, '1' );
+					$this->update_ticket_sent_counter( $attendee['attendee_id'], $this->attendee_ticket_sent );
 				}
 			}
 
 			return $sent;
 		}
 
+		/**
+		 * Update the email sent counter for attendee by 1.
+		 *
+		 * @param int    $attendee_id Attendee ID.
+		 * @param string $meta_key Meta Key that stores the count.
+		 *
+		 * @since TBD
```

Comment:
```suggestion
		 *
		 * @since TBD
		 *
		 * @param int    $attendee_id Attendee ID.
		 * @param string $meta_key Meta Key that stores the count.
```